runtime-modules/content/src/benchmarks/mod.rs

```diff
@@ -1525,22 +1509,22 @@ fn setup_nft_in_english_auction<T>(
     account_id: T::AccountId,
     actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
     video_id: T::VideoId,
-    non_channel_owner: Option<T::MemberId>,
-) -> Result<(T::VideoId, T::MemberId, T::AccountId), DispatchError>
+    non_channel_owner: bool,
+) -> Result<(NftData<T>, T::MemberId, T::AccountId), DispatchError>
 where
     T::AccountId: CreateAccountId,
     T: RuntimeConfig,
 {
     let whitelist_size = Pallet::<T>::max_auction_whitelist_length();
-    let whitelisted_members = (0..(whitelist_size as usize))
+    let whitelisted_members = (1..(whitelist_size as usize))
```

Comment:
```suggestion
    let whitelisted_members = (1..=(whitelist_size as usize))
```
(otherwise the whitelist will always be 1 member short)

Reply:
Previous comment: https://github.com/Joystream/joystream/pull/4184#discussion_r960398394
Adding an assertion does not fix the issue, it's about the range being non-inclusive on the right, so:
- if `whitelist_size == 1` the range will be empty (while it should be `[1]`)
- if `whitelist_size == 2` the range will be `[1]` (while it should be `[1, 2]`)
etc.